calendar-bundle/src/Resources/contao/modules/ModuleCalendar.php

```diff
@@ -55,7 +54,7 @@ public function generate()
 		if ($request && System::getContainer()->get('contao.routing.scope_matcher')->isBackendRequest($request))
 		{
 			$objTemplate = new BackendTemplate('be_wildcard');
-			$objTemplate->wildcard = '### ' . Utf8::strtoupper($GLOBALS['TL_LANG']['FMD']['calendar'][0]) . ' ###';
+			$objTemplate->wildcard = '### ' . $GLOBALS['TL_LANG']['FMD']['calendar'][0] . ' ###';
```

Comment:
Did you remove the `strtoupper` calls intentionally?

Reply:
Yes. This is now done via CSS.